cmd/csi-node-driver-registrar/main.go

```diff
@@ -145,22 +128,10 @@ func main() {
 		klog.Error("kubelet-registration-path is a required parameter")
 		os.Exit(1)
 	}
-	// set after we made sure that *kubeletRegistrationPath exists
-	kubeletRegistrationPathDir := filepath.Dir(*kubeletRegistrationPath)
-	registrationProbePath = filepath.Join(kubeletRegistrationPathDir, "registration")
 
-	// with the mode kubelet-registration-probe
 	if modeIsKubeletRegistrationProbe() {
-		lockfileExists, err := util.DoesFileExist(registrationProbePath)
```

Comment:
I forgot if I added this function or if it was already there, there's test code for it that might be dead code too

Reply:
+1

Reply:
Yup it seems to be dead.